### PR TITLE
release 6.0.0

### LINKED ISSUE #330: Forge Module not Published
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 8

## What are you seeing
Current published forge module for this is version 5.0.1, which was released in 2022... It seems like this is still an active project/contributions were made to allow it to support Puppet 8, so wondering if a new release will be created?

## What behaviour did you expect instead
Forge release that supports Puppet 8 (and newer versions of stdlib and inifile).

### PR DESCRIPTION
**Pull request by Dan33l**
#### This Pull Request (PR) fixes the following issues

Fixes #330

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,34 @@ All notable changes to this project will be documented in this file.
...
+- puppetlabs/inifile: Allow 6.x [\#326](https://github.com/voxpupuli/puppet-hiera/pull/326) ([zilchms](https://github.com/zilchms))
+- puppetlabs/stdlib: Allow 9.x [\#325](https://github.com/voxpupuli/puppet-hiera/pull/325) ([zilchms](https://github.com/zilchms))
+- Add Redhat 8 and 9 support [\#324](https://github.com/voxpupuli/puppet-hiera/pull/324) ([zilchms](https://github.com/zilchms))
+- Add CentOs 8 and 9 support [\#323](https://github.com/voxpupuli/puppet-hiera/pull/323) ([zilchms](https://github.com/zilchms))
+- Add Debian 11 support [\#322](https://github.com/voxpupuli/puppet-hiera/pull/322) ([zilchms](https://github.com/zilchms))
+- Add Ubuntu 22.04 support [\#321](https://github.com/voxpupuli/puppet-hiera/pull/321) ([zilchms](https://github.com/zilchms))
+
+**Closed issues:**
+
+- createkeys always creates keys in ${DEFAULT}/keys [\#118](https://github.com/voxpupuli/puppet-hiera/issues/118)
```
Comment (bastelfreak) [inline CHANGELOG.md:27]: I added skip-changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,34 @@ All notable changes to this project will be documented in this file.
...
+
+**Merged pull requests:**
+
+- Add documentation and move parameter documentation from README to REFERENCE [\#318](https://github.com/voxpupuli/puppet-hiera/pull/318) ([zilchms](https://github.com/zilchms))
+- Remove legacy top-scope syntax [\#316](https://github.com/voxpupuli/puppet-hiera/pull/316) ([smortex](https://github.com/smortex))
+- Update metadata.json to include RHEL 8 support [\#307](https://github.com/voxpupuli/puppet-hiera/pull/307) ([tq-t](https://github.com/tq-t))
```
Comment (bastelfreak) [inline CHANGELOG.md:33]: Added enhancement

File: metadata.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-hiera",
-  "version": "5.0.2-rc0",
+  "version": "6.0.0",
```
Comment (bastelfreak) [inline metadata.json:3]: Before we do the release, can you also remove the EoL operating systems listed above, in another PR?

Comment (Dan33l) [reply metadata.json:3]: @bastelfreak yes PR  #335 about this

Comment (bastelfreak) [reply metadata.json:3]: CentOS 8 is EoL as well. Can you please also drop it?

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**